src/processes/follow-topic-process.js

```diff
@@ -70,24 +72,25 @@ const sendMessageAsAnonymous = async (serverClient, communityMessageFormat, data
     created_by_id: communityMessageFormat.user_id
   });
   const createdChannel = await newChannel.create();
-  const channelState = await newChannel.watch();
+  // const channelState = await newChannel.watch();
 
   await updateBetterChannelMembers(newChannel, createdChannel, true, {
     channel_type: CHANNEL_TYPE_ANONYMOUS
   });
 
-  if (channelState.messages.length === 0) {
-    await newChannel.sendMessage({
-      text: communityMessageFormat.message,
-      user_id: communityMessageFormat.user_id
-    });
+  // if (channelState.messages.length === 0) {
+  await newChannel.sendMessage({
+    text: communityMessageFormat.message,
+    user_id: communityMessageFormat.user_id,
+    is_auto_message: true
+  });
 
-    try {
-      // await chat.stopWatching();
-    } catch (error) {
-      console.log('::: Error on stopWatching :::', error);
-    }
+  try {
+    // await chat.stopWatching();
+  } catch (error) {
+    console.log('::: Error on stopWatching :::', error);
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code block that watches the channel and sends a message if there are no messages in the channel state has been commented out. If this functionality is not needed, consider removing these lines entirely to improve code readability and maintainability. Also, the error handling for `chat.stopWatching()` is still present but commented out. If this functionality is not needed, it should be removed as well.
```diff
  // const channelState = await newChannel.watch();
 
   await updateBetterChannelMembers(newChannel, createdChannel, true, {
     channel_type: CHANNEL_TYPE_ANONYMOUS
   });
 
  // if (channelState.messages.length === 0) {
  await newChannel.sendMessage({
    text: communityMessageFormat.message,
    user_id: communityMessageFormat.user_id,
    is_auto_message: true
  });
 
  try {
    // await chat.stopWatching();
  } catch (error) {
    console.log('::: Error on stopWatching :::', error);
   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->